lnd.go

```diff
@@ -1255,7 +1255,9 @@ func genMacaroons(ctx context.Context, svc *macaroons.Service,
 	if err != nil {
 		return err
 	}
-	if err = ioutil.WriteFile(admFile, admBytes, 0600); err != nil {
+	// 640 is OK because all users have their own group by default.
+	// See commit message for more in-depth explanation.
+	if err = ioutil.WriteFile(admFile, admBytes, 0640); err != nil {
```

Comment:
We could extract the value into a constant and add the explanation from the commit message to it. Something like `adminMacUmask` or whatever. Because the commit message could be buried over time if someone updates/refactors this line.

Reply:
Also, the commit message should contain a prefix of the package and/or file(s) that were touched in the commit.